libs/ui-watt/src/lib/components/validation-message/watt-validation-message.component.scss

```diff
@@ -0,0 +1,40 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@use '@energinet-datahub/watt/utils' as watt;
+
+:host {
+  align-items: center;
+  border-radius: 4px;
+  display: flex;
+  @include watt.space-inset-squish-s;
+
+  &.watt-validation-message--info {
+    background-color: var(--watt-color-state-info-light);
```

Comment:
Question: wouldn't the correct thing todo here not setting the color to the "contrast / text color" ex. `--watt-color-state-info-contrast`?

Reply:
Good point. I forgot about them. Let's wait until UX review the colors in the design system and make the necessary adjustments, if any, afterwards.